src/dll_load_error.h

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+std::string DetermineLoadLibraryError(const char* filename);
```

Comment:
Should this be inside the `Generators` namespace?

Reply:
Hmm, good point. It's not really Generators code, and now that we're shared library only the namespace is actually redundant (just makes all of our symbol names longer while debugging).

Reply:
The JSON code isn't in the Generators namespace either, so I'll leave it outside.